website/docs/r/cos_bucket.html.markdown

```diff
@@ -59,6 +90,14 @@ The following arguments are supported:
 * `single_site_location` - (Optional,string) Location if single site bucket is desired. Accepted values: 'ams03', 'che01', 'hkg02', 'mel01', 'mex01', 'mil01', 'mon01', 'osl01', 'sjc04', 'sao01', 'seo01', 'tor01' Conflicts with: `region_location`, `cross_region_location`
 * `region_location` - (Optional,string) Location if regional bucket is desired. Accepted values: 'au-syd', 'eu-de', 'eu-gb', 'jp-tok', 'us-east', 'us-south' Conflicts with: `single_site_location`, `cross_region_location`
 * `cross_region_location` - (Optional,string) Location if cross regional bucket is desired. Accepted values: 'us', 'eu', 'ap' Conflicts with: `single_site_location`, `region_location`
+* `allowed_ip` - (Optional, list of strings) List of IPv4 or IPv6 addresses in CIDR notation to be affected by firewall in CIDR notation is supported. 
+* `activity_tracking` - (Optional, array) Enables sending log data to Activity Tracker and LogDNA to provide visibility into object read and write events.
+* `activity_tracking.read_data_events` - (Optional,bool) If set to true, all object read events (i.e. downloads) will be sent to Activity Tracker.
```

Comment:
check the activity_tracking.read_data_events,... all three should be nested inside activity_tracking